internal/tofu/context_plan.go

```diff
@@ -694,6 +699,7 @@ func (c *Context) planGraph(config *configs.Config, prevRunState *states.State,
 			ExternalReferences: opts.ExternalReferences,
 			ImportTargets:      opts.ImportTargets,
 			GenerateConfigPath: opts.GenerateConfigPath,
+			EndpointsToRemove:  endpointsToRemove,
```

Comment:
To be fair, just to stay consistent with the remaining code here (specifically ImportTargets) I would propose moving endpointsToRemove into opts.EndpointsToRemove and initializing it in the plan function in line 303, similar to how imports are prepared there.

Reply:
Done ✅